internal/command/migrate_to_v2/volumes.go

```diff
@@ -38,15 +38,14 @@ for updates, watch https://community.fly.io for announcements about volume migra
 }
 
 func (m *v2PlatformMigrator) migrateAppVolumes(ctx context.Context) error {
-
 	// NOTE: appconfig.Volume doesn't support the rare "processes" key on mounts
 	//       this is extremely rarely used, and seemingly undocumented,
 	//       but because of this rollback stores a copy of the old appconfig now
 	//       and deploys the original config if something goes wrong.
 	//       That said, once an app gets moved to V2, that mapping gets wiped.
 	// (not an issue now, because we don't even support multiple volumes on v2,
 	//  but it's worth documenting here nonetheless)
```

Comment:
this comment might be (partially) outdated now?

Reply:
looks like so but it should be working the same as before. I didn't stop to look what it was truly doing to be honest

Reply:
So, this is 1am brain, so absolutely don't take my word for this, but I think because of the interactions between the config flattening stuff you did and adding the `processes` field to mounts, this might Just Work:tm: now? Not sure, I can see about it on monday when I start finishing up all the volume stuff :)